Sources/TuistGraph/Models/MergedBinaryType.swift

```diff
@@ -0,0 +1,34 @@
+/// Represents the different options to configure a target for mergeable libraries
```

Comment:
nit: we can probably remove the documentation here and just keep the one in ProjectDescription